bin/swagger.js

```diff
@@ -0,0 +1,25 @@
+const swaggerAutogen = require('swagger-autogen')();
+
+const doc = {
+  info: {
+    title: "Inventory API",
+		description: "API for inventory management",
+		version: "1.0.0",
+		contact: {
+			name: "Allen Domingo"
+		}
+  },
+  host: 'localhost:3000',
+  schemes: ['http'],
```

Comment:
For models, we can insert a definitions object here that imports from a definitions directory
```suggestion
  schemes: ['http'],
  definitions: {
    ...supplierDefinition,
    ...customerDefinition
  }
```